### DISCUSSION TITLE
Any sample repo for testing api functions in jest (react)

### DISCUSSION THREAD

**Discussion by ShivamJoker** [sindresorhus/ky#327] [Q&A] [answered]
The library is not working with jest at all I tried ky universal too but again getting the import issue.
Please help

**Comment by sholladay** [comment, marked as answer]
See @robcaldecott's [comment](https://github.com/sindresorhus/ky/issues/170#issuecomment-762119660) where they provided a working sample repo:
https://github.com/robcaldecott/ky-jest
